src/PanoImageRenderer/PanoImageRenderer.js

```diff
@@ -367,24 +360,10 @@ export default class PanoImageRenderer extends Component {
 	_initShaderProgram(gl) {
 		const vertexShaderSource = this._renderer.getVertexShaderSource();
 		const vertexShader = WebGLUtils.createShader(gl, gl.VERTEX_SHADER, vertexShaderSource);
-
-		if (!vertexShader) {
-			return false;
-		}
-
 		const fragmentShaderSource = this._renderer.getFragmentShaderSource();
 		const fragmentShader = WebGLUtils.createShader(gl, gl.FRAGMENT_SHADER, fragmentShaderSource);
-
-		if (!fragmentShader) {
-			return false;
-		}
-
 		const shaderProgram = WebGLUtils.createProgram(gl, vertexShader, fragmentShader);
 
-		if (!shaderProgram) {
-			return null;
-		}
-
 		gl.useProgram(shaderProgram);
```

Comment:
@happyhj I wonder why you remove defense block.

Reply:
I removed because there is no case for reaching these conditions with out use case.